industrial_ci/src/util.sh

```diff
@@ -172,12 +172,16 @@ function ici_enforce_deprecated {
 
 function ici_retry {
   local tries=$1; shift
+  local ret=0
 
   for ((i=1;i<=tries;i++)); do
     "$@" && return 0
+    ret=$?
     sleep 1;
   done
-  error "'$*' failed $tries times"
+
+  ici_color_output ${ANSI_RED} "'$*' failed $tries times"
+  return $ret
```

Comment:
https://github.com/ros-industrial/industrial_ci/pull/352/files#r251236099

> There is a change though: now a failed update will stop the test and not just print a warning..

I might be misunderstood, but doesn't this line and the subsequent code prevent the ICI from failing?

Reply:
@ipa-mdl This is me just trying to stay on the safest side. If my understanding above is wrong then we can go ahead (if so please merge) as CI is passing.

Reply:
This line returns a non-zero code, which is treated as error in the calling code.